Tests/NavigationTests/NavigationAuthChallengeTests.swift

```diff
@@ -295,3 +294,7 @@ class NavigationAuthChallengeTests: DistributedNavigationDelegateTestsBase {
     }
 
 }
+
+// swiftlint:enable unused_closure_parameter
+// swiftlint:enable opening_brace
+// swiftlint:enable trailing_comma
```

Comment:
We're generally quite bad at reenabling these rules, especially in tests, does swift lint catch them itself?

Reply:
It does now tell us when you blanket disable a rule and don't re-enable it, yep - it's quite nice actually